src/status_im/chat/models.cljs

```diff
@@ -111,6 +111,10 @@
              (not (contains? chats current-chat-id)))
     (navigation/navigate-back cofx)))
 
+(fx/defn accept-activities
+  [cofx chats]
+  {:dispatch-n (map #(vector :accept-all-activity-center-notifications-from-chat %) chats)})
```

Comment:
Hi @bitgamma,

I was wondering about
`{:dispatch-n (map #(vector :accept-all-activity-center-notifications-from-chat %) chats)})`

will this call `json-rpc/call-ext-method acceptActivityCenterNotifications` and `:sanitize-messages-and-process-response` for all chats?

Wouldn't it will be better to have a single `:accept-all-activity-center-notifications-from-chats` event ? 🤔

Reply:
Indeed. I don't think there will be many chat id in a single notification, but it is probably better to consolidate it into a single rpc call